### PR TITLE
fix: fix no-data style and text align of number

### PR DESCRIPTION
**Pull request by song2kim**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [X] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [X] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [X] `Error / Warning / Lint / Type`

### Description
* change left text-align to right text-align of number
* fix no data style in BudgetUsageSummary

### CODE REVIEW

File: src/services/dashboards/widgets/_base/base-pie/BasePieWidget.vue [hunk 1]
```diff
@@ -102,8 +102,17 @@ const state = reactive({
...
+                name: state.groupBy,
+                label: groupByLabel,
+                textOptions: { type: 'reference', referenceType },
+            },
+            {
+                label: 'Cost',
+                name: 'usd_cost_sum',
+                textOptions: { type: 'cost' },
+                textAlign: 'right',
+            },
```
Comment (WANZARGEN) [inline src/services/dashboards/widgets/_base/base-pie/BasePieWidget.vue:115]: praise: 👍

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Review by minseolee — APPROVED** [review]
LGTM~!

**Review by sulmoJ — APPROVED** [review]
LGTM!!!

**Pull request merged**